frontend/pages/hosts/details/cards/Software/SelfService/SelfService.tests.tsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { screen } from "@testing-library/react";
 
+import { noop } from "lodash";
```

Comment:
Note: All this `SelfService/` directory work is just rebased off a blocking PR that's not merged yet (#27347) That UI isn't actually going to be seen because it was delayed and will be deleted. `SelfServiceNew` is the correct directory that is showing the UI for this feature and will replace that directory.